### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by subinakallyani**

### CODE REVIEW

File: src/js/controller.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const timeout = function (s) {
 async function showRecipe() {
   try {
     const id = window.location.hash.slice(1);
-    console.log(id);
+    // console.log(id);
```
Comment (syedali042) [inline src/js/controller.js]: commented logs are not needed in production, you should have that only while debugging something.

File: src/js/controller.js [hunk 2]
```diff
@@ -217,7 +217,7 @@ searchContainer.addEventListener('submit', async e => {
   e.preventDefault();
   spinner(searchResults);
   await searchDataResult(searchBox.value);
-  console.log(e, 'sdf', state.search.results);
+  // console.log(e, 'sdf', state.search.results);
```
Comment (syedali042) [inline src/js/controller.js:220]: commented logs are not needed in production, you should have that only while debugging somethin

File: src/views/searchResults.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import icons from '../img/icons.svg';
```
Comment (syedali042) [inline src/views/searchResults.js:1]: use aliases instead manual directory shift.

File: src/views/searchResults.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+  searchResultDisplay(state.search.results);
+}
+function previousButtonHandler() {
+  pageNumber = pageNumber - 1;
+  buttonStateHandler();
+
+  searchResultDisplay(state.search.results);
+}
+function buttonStateHandler() {
```
Comment (syedali042) [inline src/views/searchResults.js:72]: You can take these pagination related actions from some utils for better readability & consistency.

File: src/views/searchResults.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  searchResults.innerHTML = '';
+  searchResults.insertAdjacentHTML('afterbegin', displayItem);
+
+  searchBox.value = '';
+}
+
+nextBtn.addEventListener('click', nextButtonHandler);
+previousBtn.addEventListener('click', previousButtonHandler);
+
+function nextButtonHandler() {
```
Comment (syedali042) [inline src/views/searchResults.js:59]: You can take these pagination related actions from some utils for better readability & consistency.

**Review by syedali042 — CHANGES_REQUESTED** [review]

**Comment by syedali042** [comment]
Use Github commits conventions e.g. feat, refactor, fix, styles, chore, docs & etc. for better PR identification,

**Comment by syedali042** [comment]
@subinakallyani Put some description about the PR so reviewer can understand what's happening in this PR before he goes into code. This will be helpful for better review.

File: src/js/controller.js (hunk 1 above)
Comment (pranavmappoli) [reply src/js/controller.js]: @syedali042 How you identified this PR bro ?

File: src/views/searchResults.js (hunk 3 above)
Comment (subinakallyani) [reply src/views/searchResults.js:1]: I didn't understand. what do you mean by this?

File: src/views/searchResults.js (hunk 5 above)
Comment (subinakallyani) [reply src/views/searchResults.js:59]: Will change it later

**Review by pranavmappoli — APPROVED** [review]

**Comment by subinakallyani** [comment]
@syedali042 Thanks for the review.

**Pull request merged**